frontend/components/panels/StakePanel.tsx

```diff
@@ -19,6 +20,14 @@ export function StakePanel({ mainStakeAccount }: StakePanelProps) {
   const { data: pythBalance, isLoading: isPythBalanceLoading } =
     usePythBalance()
 
+  const { data: vestingAccountState } = useVestingAccountState(mainStakeAccount)
+
+  const accountWithLockedTokens =
+    vestingAccountState === undefined
```

Comment:
This seems a bit inconsistent with mainStakeAccount being null if it doesn't exist.
vestingAccountState is undefined if mainStakeAccount is null?

Reply:
yes

Reply:
I have made it more readable by adding the condition in that hook itself.